### DISCUSSION TITLE
when export to txt files, how to remove extra space character

### DISCUSSION THREAD

**Discussion by wisebean** [dbeaver/dbeaver#11750] [Q&A]
When I export the resultset to txt files,it always has extra space character. How to remove the extra space character? Thanks!

The txt files likes this:

![捕获](https://user-images.githubusercontent.com/4716152/111291931-a3635a80-8682-11eb-90fd-95e507fad427.PNG)

**Comment by LonwoLonwo** [comment]
This is default behaviour for TXT exporter.
You can try to use export into CSV with "|" as delimiter.
